test/functional/apps/visualize/group6/_tsvb_table.ts

```diff
@@ -195,7 +195,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         });
       });
 
-      it('should display drilldown urls', async () => {
+      it('should display drilldown urls after field formatting is applied', async () => {
```

Comment:
rule checks duplicated titles. Unique title helps to find the test in case of failure.